dvc/tree/git.py

```diff
@@ -76,96 +47,86 @@ def dvcignore(self):
     def open(
         self, path, mode="r", encoding=None
     ):  # pylint: disable=arguments-differ
-        assert mode in {"r", "rb"}
-        encoding = encoding or "utf-8"
```

Comment:
This wasn't right, we need to use locale to get default encoding, same as we do in our remote fixtures. Windows tests fail because of this now :slightly_frowning_face: